docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -0,0 +1,518 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-7-installing-web-console-on-openshift_{context}"]
+
+= Installing the {ProductName} {WebName}
+
+You install the {ProductName} ({ProductShortName}) {WebName} as part of the process of installing the {ProductShortName} Operator on the OpenShift Container Platform.
```

Comment:
@sbeskin-redhat - what do you think? should we use `You install ... ` or 
```suggestion
Install the {ProductName} ({ProductShortName}) {WebName} as part of the process of installing the {ProductShortName} Operator on the OpenShift Container Platform.
```

Reply:
@anarnold97 
I would say "You can install..."